dstore.go

```diff
@@ -194,11 +112,11 @@ func (d *Datastore) Query(q dsq.Query) (dsq.Results, error) {
 	return raw, nil
 }
 
-func (d *Datastore) RawQuery(q dsq.Query) (dsq.Results, error) {
+func (d *Datastore) rawQuery(q dsq.Query) (dsq.Results, error) {
```

Comment:
I made `rawQuery` and `queryWithParams` private since you need knowledge of their internals to be able to work with their return values (and they are not part of the `Datastore` interface).